ui/js/controllers/jobs.js

```diff
@@ -220,6 +220,28 @@ treeherderApp.controller('ResultSetCtrl', [
             });
         };
 
+        $scope.triggerAllTalosJobs = function(revision) {
+            if (!window.confirm('This will trigger all talos jobs for revision ' + revision + '!\n\nDo you want to proceed?')) {
+                return;
+            }
+
+            var times = window.prompt("Enter number of instances to have for each talos job", 6);
+            while (times > 6) {
+                times = window.prompt("We do not allow instances of each talos job to be greater than 6 times. Enter again", 6);
+            }
+
+            for (var iter=1; iter <= times; iter++) {
+                ThResultSetModel.triggerAllTalosJobs($scope.resultset.id, $scope.repoName).then(function() {
+                    thNotify.send("Request sent to trigger all talos jobs " + iter + " time", "success");
+                }, function(e) {
+                    thNotify.send(
+                        ThModelErrors.format(e, "The request for 'trigger all talos jobs' failed"),
```

Comment:
i.m.o. we should match all the thNotify wordings that currently exist (for retrigger, retriggerAllPinned, retriggerAllTalos) for each success or fail condition (with the insertion of the word 'Talos' here if really needed).

eg. for [retrigger](https://github.com/mozilla/treeherder/blob/972e889374a3651d8fc14c303505b3bd5013d531/ui/plugins/controller.js#L307)

So in this case we'd have `Unable to send retriggers` or `Unable to send Talos retriggers`?